### PR TITLE
fix flappy node test

### PR DESCRIPTION
**Pull request by Raynos**
r: @jcorbin @shannili @kriskowal

### CODE REVIEW

File: node/test/examples.js [hunk 1]
```diff
@@ -93,12 +93,15 @@ test('running as_example1.js', function t(assert) {
     }
 });
 
-test('running send_test.js', function t(assert) {
+test.only('running send_test.js', function t(assert) {
```
Comment (jcorbin) [inline node/test/examples.js]: Go Fast Mode (tm)

File: node/test/examples.js [hunk 2]
```diff
@@ -93,12 +93,15 @@ test('running as_example1.js', function t(assert) {
...
     spawnExample('send_test.js', onChild);
 
     function onChild(err, stdout, stderr) {
         assert.ifError(err);
         assert.equal(stderr, '');
+
+        console.log('wat', stdout);
```
Comment (jcorbin) [inline node/test/examples.js]: Such Chatter

**Comment by jcorbin** [comment]
lgtm

**Comment by kriskowal** [comment]
:ok:

**Pull request merged**